config/config.apps.sample.php

```diff
@@ -448,6 +448,25 @@
  */
 'collabora_group' => '',
 
+/**
+ * App: S3 Primary Object Storage
+ *
+ * Possible keys: `objectstore` ARRAY
+ */
+
+/**
+ * Configure the access parameters for a particular S3 provider.
+ * The detailed configuration of that array depends on the S3 provider.
+ * This example can therefore only show the general construct.
+ * See the "S3 Compatible Object Storage as Primary Storage Location" documentation for more details.
+ */
+'objectstore' => [
+	'class' => 'Implementation\Of\OCP\Files\ObjectStore\IObjectStore',
```

Comment:
```suggestion
	'class' => 'OCA\Files_Primary_S3\S3Storage',

```

How about a real world string there? -- I see it as a fixed string. Is there a reason to consider it as a variable? The only other implementation I know of is objectstore -- which is deprecated.

Reply:
Your point is valid but his is because the original documentation about S3 primary tells this and then further refines it to the implemented string. What we do here is to describe the constructor on a general base and then reference to the s3 documentation for the details.
As for now this is the most easiest way to follow what we already have. Honestly, I do not care if this is the only implementation or not, but if that needs a change, we have to do this also on the S3 primary documentation causing more work...
Just let me know